modules/cnv-confirming-policy-updates-on-nodes.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-confirming-policy-updates-on-nodes_{context}"]
+= Confirming Policy updates on nodes
+
+A `NodeNetworkConfigurationPolicy` manifest describes your desired network configuration for nodes in the cluster.
+When applied, the cluster creates a `NodeNetworkConfigurationPolicy` object for the whole cluster.
+The Policy object includes your desired network configuration and the actual status of the Policy on the cluster as a whole.
```

Comment:
"status of the policy" does not say much. Maybe "status of execution of the Policy"?